src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { FC, useState, useMemo } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  errorMessage: string;
+};
+
+export const ErrorNotification: FC<Props> = ({ errorMessage }) => {
+  const [isClosed, setIsClosed] = useState(false);
+
+  useMemo(() => setTimeout(() => {
+    setIsClosed(true);
+  }, 3000), [isClosed]);
```

Comment:
you need to create setTimeout on the first render and clear timeout on unmount, use useEffect